src/element/binding.ts

```diff
@@ -74,8 +78,9 @@ export const bindOrUnbindLinearElement = (
     .getNonDeletedElements(onlyUnbound)
     .forEach((element) => {
       mutateElement(element, {
-        boundElementIds: element.boundElementIds?.filter(
-          (id) => id !== linearElement.id,
+        boundElements: element.boundElements?.filter(
+          (element) =>
+            element.type !== "arrow" || element.id !== linearElement.id,
```

Comment:
shouldn't this be `element.type === 'arrow' && element.id !== linearElement.id` since we want the arrow ids except the one which matched ?

Reply:
that function is filtering out stale linear bindings, so we wanna keep non-linear bindings left intact?

Reply:
oops got it, I misunderstood